src/transformers/modeling_rag.py

```diff
@@ -377,44 +399,67 @@ def from_pretrained_question_encoder_generator(
     Args:
         input_ids (:obj:`torch.LongTensor` of shape :obj:`(batch_size, sequence_length)`):
             Indices of input sequence tokens in the vocabulary.
-            :class:`~transformers.RagConfig`, used to initialize the model, specifies which generator to use, it also specifies a compatible
-            generator tokenizer. Use that tokenizer class to obtain the indices.
-        attention_mask (:obj:`torch.Tensor` of shape :obj:`(batch_size, sequence_length)`, `optional`, defaults to :obj:`None`):
-            Mask to avoid performing attention on padding token indices in input_ids.
+            :class:`~transformers.RagConfig`, used to initialize the model, specifies which generator to use, it also
+            specifies a compatible generator tokenizer. Use that tokenizer class to obtain the indices.
+        attention_mask (:obj:`torch.Tensor` of shape :obj:`(batch_size, sequence_length)`, `optional`):
+            Mask to avoid performing attention on padding token indices.
             Mask values selected in ``[0, 1]``:
-            ``1`` for tokens that are NOT MASKED, ``0`` for MASKED tokens.
+
+            - 1 for tokens that are **not masked**,
+            - 0 for tokens that are **maked**.
+
+            `What are attention masks? <../glossary.html#attention-mask>`__
         encoder_outputs (:obj:`tuple(tuple(torch.FloatTensor)`, `optional`)
             Tuple consists of (:obj:`last_hidden_state`, `optional`: :obj:`hidden_states`, `optional`: :obj:`attentions`)
-            `last_hidden_state` of shape :obj:`(batch_size, n_docs * sequence_length, hidden_size)` is a sequence of hidden-states at the output of the last layer of the encoder.
-            `doc_scores` of shape :obj:`(batch_size, n_docs)` store retrieval scores of documents retrieved for each input in the batch.
+            :obj:`last_hidden_state` of shape :obj:`(batch_size, n_docs * sequence_length, hidden_size)` is a sequence
+            of hidden states at the output of the last layer of the encoder.
+
+            :obj:`doc_scores` of shape :obj:`(batch_size, n_docs)` store retrieval scores of documents retrieved for
```

Comment:
@patrickvonplaten This part of the docstring doesn't make any sense with the above saying this is a tuple with `last_hidden_state`, optional `hidden_states` and optional `attentions` but I wasn't sure how to fix.

Reply:
It doesn't make any sense because `last_hidden_state` does not exist in the doc string above? So  here `last_hidden_state` corresponds to `generator_enc_last_hidden_state` and `attentions` -> `generator_enc_attentions` etc => would that be fine you think?

Reply:
No `last_hidden_state is mentioned in the line before`. It's `doc_scores` that is not mentioned in this pargraph. Here is the full paragraph:
```
Tuple consists of (:obj:`last_hidden_state`, `optional`: :obj:`hidden_states`, `optional`: :obj:`attentions`)
:obj:`last_hidden_state` of shape :obj:`(batch_size, n_docs * sequence_length, hidden_size)` is a sequence of hidden-states at the output of the last layer of the encoder.
:obj:`doc_scores` of shape :obj:`(batch_size, n_docs)` store retrieval scores of documents retrieved for each input in the batch.
Used by the (:class:`~transformers.RagTokenForGeneration`) model during decoding.
```

Reply:
Fixed